patches/tModLoader/Terraria/ModLoader/Mod.Internals.cs

```diff
@@ -52,33 +52,37 @@ partial class Mod
 		}
 
 		internal void UnloadContent() {
-			Unload();
-			foreach(var loadable in content.Reverse()) {
-				loadable.Unload();
+			try {
+				Unload();
```

Comment:
This will just cause an exception cascade. Failing to unload properly should still be fatal.

Reply:
hm, true. Not really worth cleaning up garbage when the process is a ticking time bomb anyway.